Automattic/wp-calypso
# Help Center: Improve clear chat button styling

Related to https://github.com/Automattic/dotcom-forge/issues/9755
Fixes https://github.com/Automattic/dotcom-forge/issues/9755

## Proposed Changes

* Improve Help Center clear chat styling.

## Why are these changes being made?


* It does not look nice

## Testing Instructions



* Use live link and add this flag `?flags=help-center-experience`
* Start a chat with Wapuu
* Click on the ellipsis on the header ( ... )
* Compare the clear chat styling from live link and prod.
* Should look like 
<img width="460" alt="image" src="https://github.com/user-attachments/assets/29b03785-5133-43e1-ab7c-7dfbf49469af">


## Pre-merge Checklist
